s3_assets_uploader.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "aws-sdk", ">= 2.0"
+  spec.add_dependency "aws-sdk-s3", "~> 1.0"
```

Comment:
Please do not use pessimistic version constraint. Specifying the upper limit rarely helps.
It should be `>= 1.0` .

Reply:
I'm OK.
I updated and rebased as https://github.com/eagletmt/s3_assets_uploader/pull/4/commits/396da6e04869475fb0173cb47982e1f61eb9f7b3 .